.github/workflows/ci.yaml

```
@@ -44,3 +44,39 @@ jobs:
               uses: "docker://oskarstark/phpstan-ga"
               with:
                   args: analyse
+
+    auto-merge:
+        name: "Auto-Merge"
+
+        runs-on: "ubuntu-latest"
+
+        needs:
+            - "php-cs-fixer"
+            - "composer-normalize"
+            - "phpstan"
+
+        if: >
+            github.event_name == 'pull_request' &&
+            github.event.pull_request.draft == false && (
+              github.event.action == 'opened' ||
+              github.event.action == 'reopened' ||
+              github.event.action == 'synchronize'
+            ) && (
+              github.actor == 'dependabot[bot]'
```

This will automatically merge _all_ pull requests opened by Dependabot. 

In case that's not what you want, you can add a condition here to verify the pull request title, perhaps:

```suggestion
              github.actor == 'dependabot[bot]' && startsWith(github.event.pull_request.title, 'Build(deps-dev)')
```

---

No we keep all, thanks!